src/value.rs

```diff
@@ -16,6 +16,7 @@ pub enum Value {
     Str(String),
     Array(Array),
     Object(Object),
+    Null,
```

Comment:
I wonder how serde decides that `None` gets serialized as this.

We should add a serde test for this
https://github.com/cobalt-org/liquid-rust/blob/master/tests/value.rs

Reply:
I honestly have no idea. I don't really get the inner-workings of serde -- but it is on my bucket list :D .

I used serde_json as the base and it [defined `Null` in their value](https://docs.serde.rs/serde_json/value/enum.Value.html) and in JSON there also is no `None` but only `null`. For TOML and YAML I am not even sure a nothing-type even exists, I think it is just represented as empty strings ...